auv_sim/auv_sim_description/taluy_sim_description/urdf/sensors.urdf.xacro

```diff
@@ -13,4 +13,6 @@
     topic_name="cameras/cam_bottom" />
   <xacro:xsens_mti_g710_sim name="${namespace}/base_link/imu" namespace="${namespace}"
     static_reference_frame="${inertial_reference_frame}" />
-</robot>
+  <xacro:altimeter_sim name="${namespace}/base_link/altimeter" parent="${namespace}/base_link"
+    position="0.2 0 -0.11" orientation="0 0 0" />
```

Comment:
In that case (described above) position will be 0 0 0, and only orientation will be provided